actionpack/test/dispatch/session/cookie_store_test.rb

```diff
@@ -292,7 +297,7 @@ def test_session_store_with_expire_after
       end
 
       # Second request does not access the session
-      time = Time.local(2008, 4, 25)
+      time = Time.local(2008, 4, 24)
```

Comment:
`cookie_body` at line 294 contains the message with expiry metadata `time = Time.local(2008, 4, 24)` five hours from this time.
At line 307, the cookie_body contains the value from 294, but its expected expiry is changed by 1 day. Consequently, the assertion fails because the time is stubbed at: `Time.local(2008, 4, 25)`.
Hope I didn't goof up again :sweat_smile: